File: AUGUST-2025/05-08-2025/p3477.py
class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        count = 0
        for i in fruits:
            for j in range(len(baskets)):
                if i <= baskets[j]:
                    count += 1
                    baskets[j] = -1
                    break
        return len(fruits) - count
                         

File: AUGUST-2025/05-08-2025/p3477.php
class Solution {

    /**
     * @param Integer[] $fruits
     * @param Integer[] $baskets
     * @return Integer
     */
    function numOfUnplacedFruits($fruits, $baskets) {
        $unplaced = 0;
        $placed = false;
        $count = count($fruits);
        for ($i = 0; $i < $count; ++$i) { 
            $placed = false;
            for ($j = 0; $j < $count; ++$j) {
                if ($baskets[$j] >= $fruits[$i]) {
                    $baskets[$j] = -1;  
                    $placed = true;
                    break;
                }
            }

            if (!$placed) {
                $unplaced++;  
            }
        }

        return $unplaced;
        
    }
}